x-pack/solutions/security/plugins/security_solution/public/flyout/ui_tests/playwright.config.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createPlaywrightConfig } from '@kbn/scout';
+
+// eslint-disable-next-line import/no-default-export
+export default createPlaywrightConfig({
+  testDir: './tests/',
+  globalSetup: require.resolve('@kbn/scout-security/src/playwright/global_setup'),
```

Comment:
you will need to replace it with https://github.com/elastic/kibana/pull/211409 merged